src/pages/liity/index.md

```diff
@@ -198,3 +200,72 @@ Kuvaa:
+- [GBFS](https://github.com/NABSA/gbfs)-standardin mukaisen rajapinnan osoitteen, josta pyöräasemien paikat ja muut tiedot saa luettua
+
+Lisäksi voi määritellä:
+- Internet-osoitteen, josta saa yleistä tietoa kaupunkipyöräpalvelusta (kaikki kieliversiot fi, en ja sv, mikäli niitä löytyy)
+- Erillisen vuokraus- tai liittymisohjesivuston osoitteen, mikäli sellainen löytyy
+- Erillisen palautusohjesivuston osoitteen, mikäli sellainen löytyy
+- Kaupunkipyörän perusvuokra-ajan pituuden ja lisäveloitusohjeen, mikäli sellainen löytyy
+
+Yllä lueteltujen ohjesivustojen avulla voi parantaa palvelun laatua. Käyttäjälle esitetään ohjesivujen linkkejä tilanteissa,
+joissa lisäohjeet saattavat olla tarpeellisia.
+
+
+### 16. Lippuvyöhykkeet
+
+Reittiopas tukee matkalipputietojen esittämistä käyttäjälle monella tavoin. Reittiehdotuksissa voidaan kertoa tarvittavan lipun tyyppi ja oletushinta,
+mikäli GTFS-data sisältää tarvittavat hintasäännöt. Ehdotukset myös näyttävät lippuvyöhykkeen vaihtumisen pitkin matkaketjua. Taustakartalle voidaan piirtää
+vyöhykkeiden aluerajat. Kun käyttäjä napauttaa karttaa, osoitetun paikan lippuvyöhyke on myös mahdollista kertoa avatulla ponnahdusikkunalla.
+
+GTFS-datan fare rules ja fare attributes -spesifikaatiot löytyvät osoitteesta [gtfs.org/reference/static](https://gtfs.org/reference/static).
+Jos datasta löytyy tarvittavat fare tiedot, niitä voi konfiguroida reittioppaaseen vastaamalla seuraaviin kysymyksiin:
+- Näytetäänkö lippuvyöhykkeet reittiehdotuksissa (kohta 1. kuvassa alla)
+- Näytetäänkö tarvittavat liput reittiehdotuksissa (2)
+- Näytetäänkö lipun oletushinta reittiehdotuksissa (3)
+- Onko olemassa Internet-osoite ohjesivuun, josta löytyy tietoa lippujen hinnoista ja ostamisesta (4)
+- Miten GTFS fare -tiedoissa määritellyt lipputyypit esitetään käyttäjälle? Tekninen tunnus voi olla tarpeen muuttaa ihmisten tuntemaan muotoon.
+
+![](./images/liput.png)
+
+Reittiehdotusten lisäksi lippuvyöhyketietoa voidaan esittää kartalla. Sitä varten tarvitaan geoJSON-muotoista dataa WGS84-koordinaatistossa.
+On suositeltavaa käyttää mahdollisimman pientä pistemäärää (alle 1000 pistettä) ja desimaalitarkkuutta (5 desimaalia), jotta palvelu pysyy kevyenä ja nopeana.
+Paras tulos saadaan, kun käytössä on kaksi hiukan erilaista dataversiota:
+
+1. Aluetarkistusdata. Siinä lippuyöhykkeet on kuvattu suljettuina polygon tai multipolygon kohteina, yksi feature per lippuvyöhyke.
+Kukin feature sisältää properties tiedoissa ominaisuuden 'Zone', jonka arvo on vyöhykkeen tunnus, esimerkiksi 'A'
+
+Esimerkki lippuvyöhykkeiden aluedatasta: [hsl-zone-areas-20190508.json](https://raw.githubusercontent.com/HSLdevcom/pelias-api/master/middleware/config/hsl_zone_areas_20190508.json)
+
+2. Vyöhykkeiden visualisointi kartalle. Lippuvyöhykkeiden rajaviivat kuvataan linestring/multilinestring geometrioina ilman kaksinkertaista piirtoa alueiden välille.
```

Comment:
Should we include an example about the line style definitions?

Reply:
I was thinking that we add standard line styles. It might be best that the style is consistent across all journey planners.

Reply:
Added some instructions about styles, too